google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/InsertAllRequest.java

```diff
@@ -124,8 +127,11 @@ public boolean equals(Object obj) {
     }
 
     /**
-     * Creates a row to be inserted with associated id. Please notice that data for fields of type
-     * {@link LegacySQLTypeName#BYTES} must be provided as a base64 encoded string.
+     * Creates a row to be inserted with associated id.
+     *
+     * <p>To ensure proper serialization of numeric data, it is recommended to supply values using a
```

Comment:
per google tech writing style, avoid passive voice; that is, "it is recommended to supply values" --> "supply values"